### PR TITLE
[IMP] website_hr_recruitment: add website menu jobs

### PR DESCRIPTION
**Pull request by arpi-odoo**
This adds a menu to published jobs in the navigation bar of the website.

Task: 3869518

### CODE REVIEW

File: addons/website_hr_recruitment/data/config_data.xml [hunk 1]
```diff
@@ -10,6 +10,12 @@
             <field name="action_id" ref="action_open_website"/>
             <field name="state">open</field>
         </record>
+        <record id="website_menu_jobs" model="website.menu">
+            <field name="name">Jobs</field>
+            <field name="url">/jobs</field>
+            <field name="parent_id" ref="website.main_menu"/>
+            <field name="sequence" type="int">59</field>
```
Comment (Flotchet) [inline addons/website_hr_recruitment/data/config_data.xml:17]:
```suggestion
            <field name="sequence">59</field>
```

The sequence is already an int by default

**Review by Flotchet — CHANGES_REQUESTED** [review]
Just a little comment otherwhise lgtm

**Review by Bertrand2 — COMMENTED** [review]
monokuma-verdict_1

**Comment by Bertrand2** [comment]
robodoo r+

**Pull request closed without merging**